### PR TITLE
fix times of afternoon test slot

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
The proposed times are just a guess. I don't know if you had you something different in mind.

### CODE REVIEW

File: general_rules/Organization.tex [hunk 1]
```diff
@@ -97,7 +97,7 @@ \subsection{Schedule}
 	\multicolumn{1}{ c }{}
 		& \multicolumn{4}{ c }{\wcell{0.5\baselineskip}{\color{gray}Lunch}}\\\cline{2-5}
 
-	\cell{Block 2\\\footnotesize(9:00 - 12:00)}
+	\cell{Block 2\\\footnotesize(14:00 - 17:00)}
```
Comment (maximest-pierre) [inline general_rules/Organization.tex]: So we are putting 2 hours for lunch and repairs and preparation

Comment (MatthijsBurgh) [reply general_rules/Organization.tex]: 1 hour is definitely to short IMO, 1,5 maybe ok, maybe to short. 2 hours definetly good.

Comment (maximest-pierre) [reply general_rules/Organization.tex]: I would push the upper limit from 17:00 to 18:00. Just to make up for lost time or a late start in the morning.

Comment (kyordhel) [reply general_rules/Organization.tex]: I'd go for a 3 hour gap to give enough time to teams to prepare and eat. Also, this adds a nice gap in which we can fit an Open Challenge for those interested.

**Pull request merged**